applications/Unity.GrantManager/src/Unity.GrantManager.Application/Attachments/ComsS3BlobProvider.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Web;
+using Unity.GrantManager.Applications;
+using Volo.Abp.BlobStoring;
+using Volo.Abp.DependencyInjection;
+
+namespace Unity.GrantManager.Attachments;
+
+public class ComsS3BlobProvider : BlobProviderBase, ITransientDependency
+{
+    private readonly IHttpContextAccessor _httpContextAccessor;
```

Comment:
Is there another ABP wrapper around this?

Reply:
will refactor this in my next user story Ticket 8392 (Assessment Attachment)